it/neo4j/src/main/java/org/apache/beam/it/neo4j/DatabaseWaitOptions.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.it.neo4j;
+
+public class DatabaseWaitOptions {
+
+  public static DatabaseWaitOption waitDatabase() {
+    return DatabaseWait.WAIT;
```

Comment:
shall we consolidate DatabaseWait, DatabaseNoWait, DatabaseWaitInSeconds as static internal class/enum of DatabaseWaitOptions. They are all pretty thin and is produced by  DatabaseWaitOptions methods here

Reply:
I initially did this and got auxiliary class usage warnings, but I realize now that's because I did not encapsulate these classes enough.
Will follow your suggestion now.

Reply:
Fixed in d969763c5d93966a8bc21194a20b8623db8c6086